docs/libraries/http/encoding.md

```diff
@@ -134,3 +134,41 @@ model User {
 </td>
 </tr>
 </table>
+
+## Numeric types ( `int64`, `decimal128`, `float64`, etc.)
```

Comment:
Do we have a plan to add linter rule to warn in-scoped numeric is marked as encode string e.g int16, or safeint.

Reply:
good point filed na issue for an azure linter https://github.com/Azure/typespec-azure/issues/1277